simapp/app.go

```diff
@@ -529,6 +532,10 @@ func NewSimApp(
 		}
 	}
 
+	if upgradePlan, err := app.UpgradeKeeper.ReadUpgradeInfoFromDisk(); err == nil && upgradePlan.Height > 0 {
```

Comment:
to avoid needing to do this, should we move this to the newkeeper in upgrade, it would help in avoiding making users need to add this code.